README.md

```diff
@@ -247,6 +247,70 @@ parameter in `job_start` method
 
 <hr>
 
+## Datasets
+
```

Comment:
more just a nice to have, but maybe it could be helpful to define what a Dataset is.

I think the Domino documentation defines it as "A Domino Dataset is a collection of files that are available in user executions as a filesystem directory" (https://docs.dominodatalab.com/en/latest/user_guide/0a8d11/datasets-overview/)

Reply:
Agreed.